tools/convert/rs-convert.cpp

```diff
@@ -34,8 +35,17 @@ int main(int argc, char** argv) try
     ValueArg<string> outputFilenameBin("b", "output-bin", "output BIN (depth matrix) file(s) path", false, "", "bin-path");
     SwitchArg switchDepth("d", "depth", "convert depth frames (default - all supported)", false);
     SwitchArg switchColor("c", "color", "convert color frames (default - all supported)", false);
+    ValueArg <string> frameNumberStart("f", "first-framenumber", "convert frames in specific frame number range - start point", false, "", "first-framenumber");
+    ValueArg <string> frameNumberEnd("t", "last-framenumber", "convert frames in specific frame number range - end point", false, "", "last-framenumber");
+    ValueArg <string> StartTime("s", "start-time", "convert frames in specific time range - start point (in seconds)", false, "", "start-time");
+    ValueArg <string> EndTime("e", "end-time", "convert frames in specific time range - end point (in seconds)", false, "", "end-time");
```

Comment:
`ignore frames whose timestamp is greater than this value (the first frame is at time 0)`